Automattic/wp-calypso
# Gutenberg Blocks: Rename o2 preset to p2

Quoting D19603-code `#394021`

> I'd originally meant to call the preset `p2` (since that's what most of us intuitively refer to, not `o2`). However, for  [automated namespacing](https://github.com/Automattic/wp-calypso/pull/26513) of generated CSS, I changed that to `o2`, since the `body` element of P2s already had an `o2` class (but not a `p2` one) -- it was simply more convenient at the time not to add another class.
> 
> However, as of #26662, we're no longer auto-namespacing CSS with block/preset names anyway. I meant to rename the preset on the Calypso set for a while (and will proceed to do so now). This phab diff is a good opportunity to also do so on the server side.

### Testing instructions

Verify that `npm run sdk -- gutenberg client/gutenberg/extensions/presets/p2/` successfully builds files in that folder's `build/` subdirectory.
